engine/src/main/resources/templates/MalwareDetection/insightsEmailBody.html

```diff
@@ -0,0 +1,610 @@
+
+      .rh-50-block {
+        width: 100%;
+        max-width: 100% !important;
+        padding-bottom: 24px;
+      }
+
+      .rh-icon-wrapper {
+        width: 30px;
+        height: 48px;
+      }
+
+      .rh-card.rh-m-set-height {
+        height: auto;
+      }
+    }
+
+    /* Dark Mode */
+    /* @media (prefers-color-scheme: dark) {
+          .rh-content .rh-content__block,
+          .rh-card {
+            background-color: #212427;
+          }
+
+          .rh-template-container,
+          .rh-template-container > tbody {
+            background-color: #3c3f42;
+            color: #ffffff;
+          }
+
+          .rh-content a,
+          .rh-footer__text a {
+            color: #73bcf7;
+          }
+
+          .rh-cta-link a {
+            color: #ffffff;
+          }
+
+          .rh-card.rh-m-danger {
+            background-color: #c9190b;
+            background: #c9190b;
+            color: #ffffff;
+          }
+
+          .rh-card.rh-m-no-danger {
+            background-color: #b8bbbe;
+            background: #b8bbbe;
+            color: #151515;
+          }
+        }
+
+        [data‐ogsc] {
+          .rh-content .rh-content__block,
+          .rh-card {
+            background-color: #212427;
+          }
+
+          .rh-template-container,
+          .rh-template-container > tbody {
+            background-color: #3c3f42;
+            color: #ffffff;
+          }
+
+          .rh-content a,
+          .rh-footer__text a {
+            color: #73bcf7;
+          }
+
+          .rh-cta-link a {
+            color: #ffffff;
+          }
+
+          .rh-card.rh-m-danger {
+            background-color: #c9190b;
+            background: #c9190b;
+            color: #ffffff;
+          }
+
+          .rh-card.rh-m-no-danger {
+            background-color: #b8bbbe;
+            background: #b8bbbe;
+            color: #151515;
+          }
+        }  */
+
+  </style>
+  |}
+</head>
+
+<body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" class="rh-body">
+<center>
+  <table align="center" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" class="rh-body-table">
+    <tr>
+      <td align="center" valign="top" class="rh-body-cell">
+        <table border="0" cellpadding="0" cellspacing="0" class="rh-template-container">
+
+          <!-- Head -->
+          <tr>
+            <td class="rh-head">
+              <table class="rh-masthead" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                <tr>
+                  <!--Red Hat logo-->
+                  <td align="center" class="rh-masthead__brand" bgcolor="#151515">
+                    <a href="https://console.redhat.com/" target="_blank" class="rh-masthead__brand-link">
+                      <img src="https://console.redhat.com/apps/frontend-assets/email-assets/logo_insights.jpg" alt="Red Hat logo" width="182" height="90" class="rh-masthead__brand-img" />
```

Comment:
Same question about the image. It may or may not require a dynamic host.

Reply:
This one is ok too